common/actions/onboardStatus/actionCreators.ts

```diff
@@ -0,0 +1,9 @@
+import * as interfaces from './actionTypes';
+import { TypeKeys } from './constants';
+
+export type TStartOnboardSession = typeof startOnboardSession;
+export function startOnboardSession(): interfaces.StartOnboardSessionAction {
+  return {
+    type: TypeKeys.START_ONBOARD_SESSION
+  };
+}
```

Comment:
A one-action redux config seems like overkill to me, I'd probably prefer something state-based for this, but I'm don't take that as gospel, open to discussion on this.

Reply:
Looks like some of it already is in state (localStorage etc.) so I think we should either go all redux, or all state.

Reply:
transitions between tabs resets the modal, lets go all redux, we could use localStorage in redux

Reply:
Fine by me, as long as it's consistently one or the other. I'd try to move some of the modal state (specifically local storage) into redux then.